packages/r/rambdax.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "rambdax",
+  "filename": "rambdax.umd.js",
+  "description": "Lightweight alternative to Ramda - extended version",
+  "homepage": "https://selfrefactor.github.io/rambdax",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/selfrefactor/rambdax.git"
+  },
+  "keywords": [
+    "rambda",
+    "lodash",
+    "ramda",
+    "util"
+  ],
+  "author": {
+    "name": "self_refactor",
```

```suggestion
    "name": "Dejan Toteff",
```
We can use the author's actual name